### ISSUE TITLE
sql: support EXPLAIN EXECUTE ...

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#15743]
It would be nice to be able to `EXPLAIN` executing a prepared statement. That is currently impossible:

```
# Welcome to the cockroach SQL interface.
# All statements must be terminated by a semicolon.
# To exit: CTRL + D.
root@:26257/> PREPARE a AS SELECT 1;
OK
root@:26257/> EXPLAIN EXECUTE a;
invalid syntax: statement ignored: syntax error at or near "EXECUTE"
EXPLAIN EXECUTE a;
        ^
```

The proximate cause of this is that `EXECUTE` isn't part of the `explainable_stmt` rule in `sql.y`. Additionally, `planner.Explain` will have to be extended to understand how to deal with prepared statements, which it is currently unequipped to do.

**Comment by cuongdo** [comment]
is this still making it into 1.1?

**Comment by jordanlewis** [comment]
This was closed by #16868 actually.
